### PR TITLE
Security-zone API module

### PR DESCRIPTION
**Pull request by that1guy15**
Module to create and update blueprint security-zones. 
Small cleanup of various module descriptions included

This PR does not support routing-policy modifications. This will be a separate module.

### CODE REVIEW

File: library/aos_blueprint_security_zone.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+
+def validate_vlan_id(vlan_id):
+    """
+    Validate VLAN ID provided is an acceptable value
+    :param vlan_id: int
+    :return: list
+    """
+    errors = []
+    if vlan_id <= 1 or vlan_id > 4094:
```
Comment (tomasocuinnegain) [inline library/aos_blueprint_security_zone.py]: This will fail for vlan_id == 1 - should be ```if vlan_id < 1 or vlan_id > 4094```

**Review by tomasocuinnegain — COMMENTED** [review]
Did a pass and ran test playbook successfully. Approved pending fixing logic error.

**Comment by that1guy15** [comment]
@tomasocuinnegain fixed.

**Review by tomasocuinnegain — APPROVED** [review]
looks good to me

**Pull request merged**